apps/chef_objects/src/chef_user.erl

```diff
@@ -347,8 +363,12 @@ update_from_ejson(#chef_user{} = User, UserEJson) ->
     end,
     Name = username_from_ejson(UserEJson),
     Email = ej:get({<<"email">>}, UserEJson),
-    ExternalAuthenticationUid = value_or_null({<<"external_authentication_uid">>}, UserEJson),
-    RecoveryAuthenticationEnabled = ej:get({<<"recovery_authentication_enabled">>}, UserEJson) =:= true,
```

Comment:
I no longer recall the details around the use case (perhaps @jkeiser can help?) but I dimly recall that they intent was a one-shot login for special circumstances.  I don't know actual usage has reflected that or not, though.